Gopkg.lock

```diff
@@ -1114,7 +1114,7 @@
     "github.com/influxdata/influxdb",
     "github.com/influxdata/influxdb/client",
     "github.com/influxdata/influxdb/client/v2",
-    "github.com/influxdata/influxdb/influxql",
```

Comment:
Why do we still have a Gopkg if we also have a `go.mod` ? Can we delete this file?

Reply:
I think we can.   I'll make a separate PR to do that.